chat-widget/src/components/postchatsurveypanestateful/PostChatSurveyPaneStateful.tsx

```diff
@@ -21,7 +21,10 @@ export const PostChatSurveyPaneStateful = (props: IPostChatSurveyPaneStatefulPro
     const generalStyleProps: IStyle = Object.assign({}, defaultGeneralPostChatSurveyPaneStyleProps, props.styleProps?.generalStyleProps,
         {display: state.appStates.isMinimized ? "none" : ""});
     let surveyInviteLink = "";
-    if (state.domainStates.postChatContext.surveyInviteLink) {
+    if (state.appStates.shouldUseBotSurvey && state.domainStates.postChatContext.botSurveyInviteLink) {
+        surveyInviteLink = state.domainStates.postChatContext.botSurveyInviteLink + "&embed=" + (postChatSurveyMode === PostChatSurveyMode.Embed).toString() + 
+        "&compact=" + (props.isCustomerVoiceSurveyCompact ?? true).toString() + "&lang=" + (state.domainStates.postChatContext.formsProLocale ?? "en") + "&showmultilingual=false";
+    } else {
         surveyInviteLink = state.domainStates.postChatContext.surveyInviteLink + "&embed=" + (postChatSurveyMode === PostChatSurveyMode.Embed).toString() + 
         "&compact=" + (props.isCustomerVoiceSurveyCompact ?? true).toString() + "&lang=" + (state.domainStates.postChatContext.formsProLocale ?? "en") + "&showmultilingual=false";
```

Comment:
Can we move these strings in the links creation to constants file since its being used at multiple places and should be the same too.?

Reply:
Ideally these strings are url links for PCs and can differ based on the props configured and config settings. I do not think moving it to the constants file might be the best way to go.
And in future both can be different since one handles url creation for bot surveys and another for all other regular scenarios

Reply:
Thanks for clarifying.